docs/run.sh

```diff
@@ -204,6 +207,11 @@ BootstrapLinux() {
     # flaky connection to Launchpad PPAs.
     Retry sudo apt-get update -qq
 
+    # Apr 2021: For R 3.5 API first remove installed R version which may be 4.0 
+    if [[ "${R_VERSION}" == "3.5" ]]; then
+        sudo apt-get remove -y r-base-core
+    fi
+
```

Comment:
Hm.  An R testing framework that removes R?  Where would we get 3.6.3 from, reliably?  That is still the package, albeit in a different repo.  So maybe we want to switch repos instead?

Reply:
The problem is that r-base is already installed in version 4.0.5 when you start a Ubuntu 18.04 container on GitHub Actions, so I don't see how we can get an older version without first removing this. It is documented under tools in the Ubuntu 18.04 README:
https://github.com/actions/virtual-environments/blob/ubuntu18/20210405.1/images/linux/Ubuntu1804-README.md#tools

Reply:
Yes. But as discussed, that is for other reasons.  Please remove the five lines 210 to 214 above.